kentik_api_library/kentik_api/public/device_label.py

```diff
@@ -43,29 +44,54 @@ def __init__(
 
     # pylint: enable=too-many-arguments
 
+    @classmethod
+    def new(cls, name: str, color: str):
+        return cls(
+            name=name,
+            color=color,
+            devices=[],
+            id=DEFAULT_ID,
+            user_id=DEFAULT_ID,
+            company_id=DEFAULT_ID,
+            created_date=DEFAULT_DATE,
+            updated_date=DEFAULT_DATE,
+        )
+
+    @classmethod
+    def update(cls, id: ID, name: str, color: str):
```

Comment:
```suggestion
    def new_for_update(cls, id: ID, name: str, color: str):
```
This makes it more obvious that it creates new object.